x-pack/plugins/data_visualizer/public/plugin.ts

```diff
@@ -67,6 +69,10 @@ export class DataVisualizerPlugin
     if (plugins.embeddable) {
       registerEmbeddables(plugins.embeddable, core);
     }
+
+    if (plugins.share) {
```

Comment:
`share` is a required plugin (as is `embeddable` above) and so we don't need this check to see if it's undefined.

Reply:
Fixed here https://github.com/elastic/kibana/pull/121053/commits/a2359f1bc1e9630f67965f07fc3b7f667dfc0ba4